index.html

```diff
@@ -49,5 +49,159 @@ <h4 class="h4-connect">LET'S CONNECT</h4>
+          <li><img src="/image/Tag1.png" alt="css" /></li>
+          <li><img src="/image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <button>See Project</button>
+      </div>
+
+      <!-- fourth work section -->
+
+      <div class="card">
+        <img class="img" src="/image/Snapshoot4.png" alt="ID" />
+        <div class="card-contents"></div>
+        <h2>Mult-Post Stories</h2>
+        <ul class="works">
+          <li>CANOPY</li>
+          <li><img src="/image/counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="/image/counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="works2">
+          <li><img src="/image/Tag.png" alt="html" /></li>
+          <li><img src="/image/Tag1.png" alt="css" /></li>
+          <li><img src="/image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </section>
+<div class="bg-white">
+    <section id="aboutme">
+      <div class="me">
+        <h3 class="aboutme">About Myself</h3>
+        <p class="p">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+
+        <h4 class="h4-connect">LET'S CONNECT</h4>
+
+        <ul class="flex item-center">
+          <li class="icon-tag">
+            <a href="#"><img src="/image/twitter.png" alt="twitter" /></a>
+          </li>
+          <li class="icon-tag">
+            <a href="#"><img src="/image/Linkedin.png" alt="linkedin" /></a>
+          </li>
+          <li class="icon-tag">
+            <a href="#"><img src="/image/m.png" alt="m" /></a>
+          </li>
+          <li class="icon-tag">
+            <a href="#"><img src="/image/git.png" alt="Github" /></a>
+          </li>
+          <li class="icon-tag">
+            <a href="#"><img src="/image/angelist.png" alt="angelist" /></a>
+          </li>
+        </ul>
+        <button>Get my resume</button>
+      </div>
+      <div class="base-about-myself">
+      <h4>Languages</h4>
+      <ul>
+        <li class="flex item-center bg-languages"><a><img src="/image/ellipsejs.png" alt="frameJs" /></a>JavaScript</li>
+        <li class="flex item-center bg-languages"><a><img src="/image/ellipsehtml.png" alt="frameJs" /></a>HTML</li>
+        <li class="flex item-center bg-languages"><a><img src="/image/ellipsecss.png" alt="frameJs" /></a>CSS</li>
+       
+      </ul>
+      <div>
+        <ul>
+          <li class="frameworks"><p>Frameworks</p> <p>></p></li> <hr>
+          <li class="skills"><p>Skills</p> <p>></p></li>
+          <hr>
+        </ul>
+       
+      </div>
+    </div>
```

Comment:
**[OPTIONAL]** - There are some little differences between your design and the Figma design. Please try to fix this 🙏 . It is important to replicate exact design specs so recruiters can be confident in your ability to produce exact design templates.

Your project currently looks like this ( Please pay attention to the alignment of the arrows):
![skills current project](https://user-images.githubusercontent.com/73607512/192119369-aac7046e-597a-4a9b-8df6-b48ea42b1ec3.PNG)

Figma template:
![skills figma](https://user-images.githubusercontent.com/73607512/192119397-7e5fd1e0-7fe9-4da5-9cb3-6c72da8a675a.PNG)